content/10.methods.md

```diff
@@ -91,6 +91,10 @@ We used the estimated probabilities and did not convert to a hard group assignme
 For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of .74 and a probability of female of 0.26, which we would add for an author with this first name.
 Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as individual values for specific scientists.
 
+Excluding 45 authors with empty fore name field, genderize.io fails to provide gender predictions for 3,044 fore names of 34,005 corresponding authors, and two names of 411 ISCB honorees.
+Of these names, approximately 45% have no more than three characters which are likely initials (e.g., "AB", "A.B", or "A B"), and 27% have a hyphen and more than three characters.
```

Comment:
I think this description is different than our processing pipeline. A name like "AB" would be set to None in the simplify_fore_name step. Therefore, we never make a genderize call for the name. I imagine the following categories:

1. author does not have fore name (i.e. missing metadata)
2. author has fore name that is None once simplified (i.e. initials)
3. author has fore name that genderize cannot predict
4. author has fore name that genderize can predict

We should differentiate 2 and 3 (but it may be okay to group 1 and 2 together).